jedi_vim.py

```diff
@@ -469,8 +469,12 @@ def new_buffer(path, options=''):
             'top': 'topleft split',
             'left': 'topleft vsplit',
             'right': 'botright vsplit',
-            'bottom': 'botright split'
+            'bottom': 'botright split',
+            'winwidth': 'vs'
         }
+        if user_split_option == 'winwidth' and vim.current.window.width <= 2 * int(vim_eval("&tw ? &tw : 80")):
+            for key, value in split_options.items():
```

Comment:
Fair enough, just add another commit. Please also rename `&tw` to `&textwidth`. I don't like the shortcuts in VIM script, they make everything way harder to read. (I know the `textwidth` option very well, but didn't know what it meant at first.)